src/open_inwoner/openzaak/api_models.py

```diff
@@ -137,10 +137,52 @@ class Resultaat(ZGWModel):
     toelichting: Optional[str] = ""
 
 
+@dataclass
+class StatusType(ZGWModel):
+    url: str  # bug: not required according to OAS
+    zaaktype: str
+    omschrijving: str
+    volgnummer: int
+    omschrijving_generiek: str = ""
+    statustekst: str = ""
+    is_eindstatus: bool = False
+    # not in eSuite
+    informeren: Optional[bool] = False
+
+
 @dataclass
 class Status(ZGWModel):
     url: str
     zaak: Union[str, Zaak]
     statustype: Union[str, StatusType]
     datum_status_gezet: Optional[datetime] = None
     statustoelichting: Optional[str] = ""
+
+
+@dataclasses.dataclass
```

Comment:
Is it different from the one used above?It seems to be the same import.

Reply:
Not really, just sloppy use of auto-imports. Fixed it.